userdocs/src/usage/vpc-cluster-access.md

```diff
@@ -68,6 +68,10 @@ To update the restrictions using a `ClusterConfig` file, set the new CIDRs in `v
 eksctl utils set-public-access-cidrs -f config.yaml
 ```
 
+!!!warning
+    If setting `publicAccessCIDRs` and creating nodegroups `privateAccess` should be set to `true` to avoid
+    `context deadline exceeded` errors.
```

Comment:
This should properly explain why the context deadline exceeds. Also, please link the created issue that you closed previously. We usually don't close issues by hand if they require a PR. We let github do its thing. :) 
https://github.com/weaveworks/eksctl/issues/4569

Reply:
@Skarlso done : )

Also sorry about closing the other PR.